src/redux/actions/article/index.js

```diff
@@ -0,0 +1,33 @@
+import { toast } from 'react-toastify';
+import * as actions from '../actionTypes';
+
+const token = localStorage.getItem('token');
+
+export const successCreateArticleAction = (data) => ({
+  type: actions.CREATE_ARTICLE_SUCCESS,
+  payload: data,
+});
+export const failedCreateArticleAction = (error) => ({
+  type: actions.CREATE_ARTICLE_FAIL,
+  payload: error,
+});
+export const CreateArticleAction = (input) => (dispatch) => fetch('http://localhost:8000/api/v1/articles', {
```

Comment:
I think it would be good to use the path variables from action types to construct the fetch URL

Reply:
at the moment we are using localhost since a fixed bug in our back-end is not yet merged,@leon will change it then I can cal heroku url instead